packages/cozy-stack-client/src/TriggerCollection.spec.js

```diff
@@ -85,17 +158,42 @@ describe('TriggerCollection', () => {
   })
 
   describe('find', () => {
-    it('should throw if no worker in selector', async () => {
-      await expect(collection.find()).rejects.toThrowError()
-    })
-
-    it('should call /jobs/triggers route', async () => {
+    it('should call /jobs/triggers route if only worker selector is passed', async () => {
       await collection.find({ worker: 'thumbnail' })
       expect(stackClient.fetchJSON).toHaveBeenLastCalledWith(
         'GET',
         '/jobs/triggers?Worker=thumbnail'
       )
     })
+
+    it('should call /data/io.cozy.triggers/_find route if multiple arguments are passed', async () => {
+      stackClient.fetchJSON.mockReturnValueOnce(
+        Promise.resolve({
+          id: '_design/123456',
+          name: '123456',
+          result: 'exists'
+        })
+      )
+      stackClient.fetchJSON.mockReturnValue(FIND_RESPONSE_FIXTURES)
```

Comment:
Since we're not using the `FIND_RESPONSE_FIXTURES ` in the test, do we really need them?

Reply:
Yep since we're doing all the work in `.find()` method below, we need to return values with the right attributes. Otherwise, it'll throw an error I think.